runtime/ops/os/mod.rs

```diff
@@ -341,27 +342,127 @@ fn op_uid(
   Ok(None)
 }
 
-// HeapStats stores values from a isolate.get_heap_statistics() call
-#[derive(Serialize)]
-#[serde(rename_all = "camelCase")]
-struct MemoryUsage {
-  rss: usize,
-  heap_total: usize,
-  heap_used: usize,
-  external: usize,
+#[op2]
+#[serde]
+fn op_runtime_cpu_usage() -> (usize, usize) {
```

Comment:
Most of the other ops return a Struct, not just a tuple. While a tuple may be faster (I'm not sure how much performance matters here) consistency might be better.